prow/github/helpers_test.go

```diff
@@ -131,3 +131,90 @@ func TestPermissionsFromLevel(t *testing.T) {
 		}
 	}
 }
+
+func TestMatchCardToContent(t *testing.T) {
+	var testCases = []struct {
+		card       ProjectCard
+		contentURL string
+		match      bool
+	}{
+		{
+			// match issue url to the content url in the card
```

Comment:
nit: we add this as a `name` field to the test case so we can emit it on failures and make it easier to understand what happened in the test output

Reply:
I added name field, I am not able to address the issue with the match though. Either I always expect that the url of the object is passed but in most cases we use html_url and not the url. I am trying to handle both url and html_url and they are different. Parsing URL would not be deterministic. If we decide that we should not handle html_url then it will be very simple comparison which is the first thing we do in this method. If that fails we are trying to match html_url. Please let me know if there is a better way to address this.
I would like to resolve this ASAP.